File: CS/Form1.cs
using System;
using System.Windows.Forms;
// ...

namespace RepPGVertHeadersHeight {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            pivotGridControl1.Fields["MyRow"].BestFit();
        }

        private void button2_Click(object sender, EventArgs e) {
            XtraReport1 report = new XtraReport1();
            //new DevExpress.XtraReports.UI.ReportDesignTool(report).ShowDesignerDialog();
            new DevExpress.XtraReports.UI.ReportPrintTool(report).ShowPreviewDialog();
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
' ...

Namespace RepPGVertHeadersHeight
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			pivotGridControl1.Fields("MyRow").BestFit()
		End Sub

		Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button2.Click
			Dim report As New XtraReport1()
			'new DevExpress.XtraReports.UI.ReportDesignTool(report).ShowDesignerDialog();
			CType(New DevExpress.XtraReports.UI.ReportPrintTool(report), DevExpress.XtraReports.UI.ReportPrintTool).ShowPreviewDialog()
		End Sub
	End Class
End Namespace